pallets/committee-management/src/impls.rs

```diff
@@ -423,6 +437,63 @@ impl<T: Config> Pallet<T> {
             Self::deposit_event(Event::BanValidators(fresh_bans));
         }
     }
+
+    /// Calculates committee for the given session.
+    /// If the era `E` starts in the session `a`, and ends in session `b` then from session `a-1`
+    /// to session `b-1` this function can answer question who will be in the committee in the era `E`.
+    /// In the last session of the era `E` this can be used to determine all of the sessions in the
+    /// era `E+1`.
+    pub fn session_committee_for_session(
+        session: SessionIndex,
+    ) -> Result<SessionCommittee<T::AccountId>, SessionValidatorError> {
+        let ae = match T::EraInfoProvider::active_era() {
+            Some(ae) => ae,
+            _ => return Err(SessionValidatorError::Other("No active era".encode())),
+        };
+
+        let current_session = T::SessionInfoProvider::current_session();
+
+        let active_starting_index = match T::EraInfoProvider::era_start_session_index(ae) {
+            Some(asi) => asi,
+            // Shouldn't happen
+            None => {
+                return Err(SessionValidatorError::Other(
+                    "No known starting session for current era".encode(),
+                ))
+            }
+        };
+
+        // Historical session
+        if session < active_starting_index - 1 {
```

Comment:
This is most likely off-by-one and should be `session < active_starting_index`...

Reply:
This is actually because at the point of the session `active_starting_index - 1` we already have everything we need to calculate the committee for the era that `active_starting_index` starts

Reply:
Yeah, but in that case I think you confused `session` with `current_session` :P But this is gone now :)

Reply:
:upside_down_face: right